### PR TITLE
Breadcrumb mango final

### PR DESCRIPTION
**Pull request by robertkowalski**
reactify former "breadcrumb"-area.

### CODE REVIEW

**Comment by robertkowalski** [comment]
depends on the spinner.js PR

**Comment by garrensmith** [comment]
+1

File: app/addons/documents/assets/less/header-docs-left.less [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  height: @collapsedNavWidth;
+  width: 247px;
+  color: #333;
+  font-size: 24px;
+  line-height: 24px;
+  padding: 22px 10px;
+  border-right: 1px solid #ccc;
+}
+
+.faux-header__doc-header-dropdown-toggle {
```
Comment (garrensmith) [inline app/addons/documents/assets/less/header-docs-left.less]: Can you change this to `button.faux-header__doc-header-dropdown-toggle {`

File: app/addons/documents/assets/less/header-docs-left.less [hunk 2]
```diff
@@ -0,0 +1,78 @@
+@import "../../../../../assets/less/variables.less";
+
+.faux-header__doc-header-left {
```
Comment (robertkowalski) [inline app/addons/documents/assets/less/header-docs-left.less]: bem

**Pull request closed without merging**